src/main/backend.ts

```diff
@@ -316,12 +335,20 @@ export class MainBackend extends BaseBackend<ElectronOptions> implements CommonB
   }
 
   /** Returns extra information from a renderer's web contents. */
-  private _getRendererExtra(contents: Electron.WebContents): { [key: string]: any } {
+  private _getRendererElectronContext(contents: Electron.WebContents, details?: Electron.Details): Event {
     const customName = this._options.getRendererName && this._options.getRendererName(contents);
-
-    return {
+    const electronContext: { [key: string]: any } = {
```

Comment:
```suggestion
    const electronContext: Record<string, any> = {
```

we can’t type this stronger? We are explicitly setting fields below.